miracl/flow/miracl_workflow_ace_parser.py

```diff
@@ -56,7 +58,7 @@ def parsefn(self) -> argparse.ArgumentParser:
             "Single or multi method arguments"
```

Comment:
Also, it might also be useful to have a note in the help to tell users that they have to choose a single or multi method argument first before choosing a required argument the reason being that the former are not explicitly marked as "required".

Reply:
Maybe sth like this: `required arguments (set single or multi method arguments first):`